spec/models/metadata_field_spec.rb

```diff
@@ -26,4 +30,55 @@
     expect { create(:metadata_field, name: 'sample_name') }.to raise_error(ActiveRecord::RecordInvalid)
     expect { create(:metadata_field, name: 'my_name') }.to_not raise_error
   end
+
+  it 'will not create a required field that is not default_for_new_host_genome' do
```

Comment:
Sometimes is good to create contexts to group tests:
```
context 'for required fields' do
	context 'if is not default for new hosts' do
		it 'will not create ' do
		it 'will not update' do
```
I feel it is easier to read as opposed to the long sentences.